core/src/main/resources/ssl/self_signed_fips/java.security.ibm-java-1.8

```diff
@@ -0,0 +1,1420 @@
+# ===========================================================================
```

Comment:
@sgala Please double check that there aren't any differences in all the java.security.XXX files as it doesn't make sense to add a load of if they are all the same.